src/main/java/org/opensearch/security/auth/RolesInjector.java

```diff
@@ -67,15 +71,17 @@ public RolesInjector() {
         Set<String> roles = ImmutableSet.copyOf(strs[1].split(","));
 
         if(user != null && roles != null) {
-            addUser(user, ctx);
+            addUser(user, transportRequest, action, task, ctx);
         }
         return roles;
     }
 
-    private void addUser(final User user, final ThreadContext threadContext) {
+    private void addUser(final User user, final TransportRequest transportRequest,
+                         final String action, final Task task, final ThreadContext threadContext) {
         if(threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER) != null)
             return;
 
         threadContext.putTransient(ConfigConstants.OPENDISTRO_SECURITY_USER, user);
+        auditLog.logSucceededLogin(user.getName(), true, null, transportRequest, action, task);
```

Comment:
Should `securityadmin` be set to `false`? AFAIK, only for certificate user it should be true.